### PR TITLE
Adding placeholder image in Video block

### PR DESCRIPTION
**Pull request by iFlameing**

### CODE REVIEW

File: src/components/manage/Blocks/Video/VideoSidebar.jsx [hunk 1]
```diff
@@ -68,21 +72,42 @@ const VideoSidebar = ({
...
+                    resetSubmitUrl();
+                    onChangeBlock(block, {
+                      ...data,
+                      url: '',
+                    });
+                  }}
+                  onChange={() => {}}
+                />
+                <TextWidget
+                  id="external"
```
Comment (tiberiuichim) [inline src/components/manage/Blocks/Video/VideoSidebar.jsx:92]: same id, external, as above

Comment (iFlameing) [reply src/components/manage/Blocks/Video/VideoSidebar.jsx]: Fixed

**Review by tiberiuichim — APPROVED** [review]

**Comment by tisto** [comment]
@iFlameing todo

- add circle to -> symbol (maybe we have a more youtube-like icon for play)
- replace image url with content browser
- nice-to-have: show preview image in edit mode

**Comment by iFlameing** [comment]
@tisto We can't pass the custom icon to the embed semantic UI components. We are storing video URL to `url` parameter, if  I implement the object browser then URL field will get overwritten. I can solve it by changing the `url` to video URL and storing the preview image at URL field.

**Review by tiberiuichim — COMMENTED** [review]
There's a way to programatically create a thumbnail URL for youtube videos. (maybe also vimeo). See https://stackoverflow.com/a/12788320/1220919

**Comment by tisto** [comment]
Maybe @sneridagh can have a look here and give his opinion if the large refactoring is necessary and worth the effort. This is still a good feature to have in Volto.

**Comment by sneridagh** [comment]
> @tisto We can't pass the custom icon to the embed semantic UI components. We are storing video URL to `url` parameter, if I implement the object browser then URL field will get overwritten. I can solve it by changing the `url` to video URL and storing the preview image at URL field.

You can choose the field to be written by the ObjectBrowser. See: https://docs.voltocms.com/blocks/editcomponent/#openobjectbrowser-handler-api

**Comment by sneridagh** [comment]
@iFlameing I think we can go if:
- You use the ObjectBrowser (using another property than url to store the URL) and allow to paste any URL too (TextLineWidget)
- The play button maybe we can place a (transparent) wrapper that has the icon in the middle centered using CSS?

**Comment by iFlameing** [comment]
@sneridagh @tisto I implemented the selecting image from the object browser and fix everything regarding showing placeholder.

One thing I am not able to solve is Icon. The semantic UI icon is broken.

**Comment by tisto** [comment]
@iFlameing pls give us a heads up on this.

**Comment by iFlameing** [comment]
@tisto done. All the functionality is there like everything. Only the icon is missing.

**Comment by sneridagh** [comment]
Now it's ready :)

**Pull request merged**